mapic/mistapiconnector_app.go

```diff
@@ -150,12 +151,12 @@ func (mc *mac) Start(ctx context.Context) error {
 		glog.Infof("AMQP url is empty!")
 	}
 	if producer != nil && mc.config.MistScrapeMetrics {
-		startMetricsCollector(ctx, statsCollectionPeriod, mc.nodeID, mc.ownRegion, mc.mist, lapi, producer, ownExchangeName, mc)
+		mc.metricsCollector = createMetricsCollector(ctx, statsCollectionPeriod, mc.nodeID, mc.ownRegion, mc.mist, lapi, producer, ownExchangeName, mc)
```

Comment:
We used to run the stats collector loop every `statsCollectionPeriod` (10s) while the `reconcileLoop` runs every 60s. I think 60s is too much for updating the `isActive` property, as it is very close to the limit at which the Studio API will reset its value assuming the stream is unused (I think it's 70s there).

To avoid making the `isActive` field going back-and-forth, wdyt of going on a middle ground here, maybe `30s`? Also make sure to delete any unused consts like this `statsCollectionPeriod`, or update its name and value to the new abstractions.

@leszko WDYT? You think reducing the reconcile loop to 30s would be too short?

Reply:
Changed it to 30s to test it in staging, will leave this conversation open for @leszko feedback